Godeps/Godeps.json

```diff
@@ -30,32 +30,32 @@
 		},
 		{
 			"ImportPath": "github.com/Azure/azure-sdk-for-go/arm/compute",
-			"Comment": "v10.0.4-beta-1-g786cc84",
+			"Comment": "v10.0.4-beta-1-g786cc841",
```

Comment:
why those pkgs are changed from g786cc84 to g786cc841?

Reply:
not sure.  the revision is still the same, so it shouldn't matter.